R/BenchmarkResult.R

```diff
@@ -99,6 +99,7 @@ BenchmarkResult = R6Class("BenchmarkResult",
     #' Printer.
     print = function() {
       tab = self$aggregate(measures = list(), conditions = TRUE)
+      setattr(tab, "class", c("data.table", "data.frame"))
```

Comment:
can you proceed without dropping the class, e.g. by calling `print.data.table()` instead of `print()`?

Reply:
Unfortunately, we cannot access `print.data.table()`. At least `settattr()` works by reference.